model-optimizer/extensions/front/AttributedRollToRoll.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from extensions.ops.roll import Roll
+from mo.front.common.replacement import FrontReplacementPattern
+from mo.graph.graph import Graph, rename_nodes
+from mo.ops.const import Const
+
+
+class AttributedRollToRoll(FrontReplacementPattern):
```

Comment:
Maybe this transformation should use `FrontReplacementOp`?

Reply:
Replaced with FrontReplacementOp.